tests/test_casereader.py

```diff
@@ -224,6 +224,13 @@ def test_case_reader_input_parameter():
     assert momentum.numeric_value == 12.4
     assert momentum.value == "12.4 [kg m s^-1]"
 
+    velocity = InputParameter(raw_data=(("name", "v"), ("definition", "2[m / s]")))
+
+    assert velocity.name == "v"
+    assert velocity.units == "m / s"
```

Comment:
The main case to test is without the spaces, as reported. We don't necessarily anticipate this scenario but if it appeared, why would we retain these spaces?

Reply:
It is resolved as per our discussion.
Thank you.